scheduler/scheduler-server/src/test/resources/functionaltests/workflow/descriptors/recovery_for.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job name="RecoveryFor" priority="normal" projectName="Not Assigned" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:dev" xmlns="urn:proactive:jobdescriptor:dev">
```

Comment:
Yes, its a copy of the latest version. Usually it is used for testing purposes or development so that all workflows do not have to be updated when the schema version is increased.